saleor/dashboard/order/forms.py

```diff
@@ -309,23 +319,16 @@ def refund(self):
         return self.try_payment_action(gateway_refund)
 
 
-class VoidPaymentForm(forms.Form):
+class VoidPaymentForm(ManagePaymentForm):
+
+    clean_status = ChargeStatus.NOT_CHARGED
```

Comment:
From above suggestion:
```suggestion
    clean_status = ChargeStatus.NOT_CHARGED,
```